src/features/components/templates/footer/description.tsx

```diff
@@ -12,7 +12,7 @@ export const DescriptionContentROW = () => {
         <>
             <Text size="sm">
                 <Localize
-                    translate_text="_t_Deriv (FX) Ltd is licensed by the Labuan Financial Services Authority <0>(licence)</0>. Deriv (BVI) Ltd is licensed by the British Virgin Islands Financial Services Commission <1>(licence)</1>. Deriv (V) Ltd is licensed by the Vanuatu Financial Services Commission <2>(licence)</2>. Deriv (SVG) LLC has a registered office at First Floor, SVG Teachers Credit Union Uptown Building, Corner of James and Middle Street, Kingstown P.O., St Vincent and the Grenadines. The holding company for these subsidiaries is Deriv.com Limited, with the registration number 71479 and the registered address at 2nd Floor, 1 Cornet Street, St Peter Port, Guernsey, GY1 1BZ._t_"
+                    translate_text="_t_Deriv (FX) Ltd is licensed by the Labuan Financial Services Authority._t_"
                     components={[
                         <a
                             className={clsx(sharedClasses)}
```

Comment:
Do we still need this? 
I can't see any usage of this in `_t_Deriv (FX) Ltd is licensed by the Labuan Financial Services Authority._t_`